.gitignore

```diff
@@ -0,0 +1,72 @@
+# OSX
```

Comment:
There are  a lot of files here that are OS specific. Those should be defined in your global `.gitignore` and not here.

Reply:
Rebased with master which already as the global .gitignore config, and also created a docsaurus scope related .gitignore.